src/main/java/com/theironyard/controllers/IronQuotesController.java

```diff
@@ -0,0 +1,127 @@
+package com.theironyard.controllers;
+
+import com.theironyard.command.QuoteCommand;
+import com.theironyard.command.TagCommand;
+import com.theironyard.entities.Quote;
+import com.theironyard.entities.Tag;
+import com.theironyard.entities.User;
+import com.theironyard.services.QuoteRepository;
+import com.theironyard.services.TagRepository;
+import com.theironyard.services.UserRepository;
+import com.theironyard.utilities.PasswordStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Created by vasantia on 8/1/16.
+ */
+
+@RestController
+public class IronQuotesController {
+
+    @Autowired
+    QuoteRepository quoteRepository;
+
+    @Autowired
+    TagRepository tagRepository;
+
+    @Autowired
+    UserRepository userRepository;
+
+    @RequestMapping(path = "/create", method = RequestMethod.POST)
```

Comment:
This should be `/quotes/create`.  Just `/create` does not clearly indicate what will be created.